plugins/inputs/mdstat/README.md

```diff
@@ -0,0 +1,41 @@
+# MDSTAT Input Plugin
+
+The `mdstat` plugin gathers metrics on raid arrays on the system that are managed by mdadm
+
+#### Configuration
+```toml
+[[inputs.mdstat]]
+  # The location of the mdstat file to read.
+  # mdstat_file = /proc/mdstat
```

Comment:
Other telegraf input plugins that read from proc look for an environment variable named HOST_PROC that allows the plugin to be used when telegraf is running in a container and the proc owned by the container host is somewhere other than /proc.  Could you add this support?  For examples, see inputs/bond, inputs/wireless, inputs/synproxy, inputs/linux_sysctl_fs.

Reply:
I went ahead and added this support. The main reason I had the mdstat_file config item was so I could throw arbitrary mdstat files at it for testing. You can do the same with the host_proc config item used in inputs/bond, so I replaced mine completely.